variables.tf

```
@@ -66,6 +66,12 @@ variable "blob_delete_retention_policy" {
   default     = 30
 }
 
+variable "file_retention_policy" {
+  description = "The number of days that files should be retained"
```

Ensure consistent punctuation in variable descriptions.

```suggestion
  description = "The number of days that files should be retained."
```